cfme/cloud/instance/openstack.py

```diff
@@ -124,15 +124,22 @@ class form(View):  # noqa
 
 
 class DetachVolumeView(CloudInstanceView):
-    @View.nested
-    class form(View):  # noqa
-        volume = BootstrapSelect('volume_id')
-        submit_button = Button('Submit')
-        cancel_button = Button('Cancel')
+    volume = BootstrapSelect('volume_id')
+    submit_button = Button('Submit')
+    cancel_button = Button('Cancel')
 
     # Only the instance name is displayed, cannot confirm provider
     is_displayed = displayed_not_implemented
 
+    def after_fill(self, was_change):
+        # TODO: BZ1647695
```

Comment:
Please add more information to this TODO like `Remove this method once BZ1647695 has been fixed`.